core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -3037,14 +3062,71 @@ class KafkaApisTest extends Logging {
   }
 
   @Test
-  def shouldAppendToLogOnWriteTxnMarkersWhenCorrectMagicVersion(): Unit = {
+  def shouldAppendToLogOnWriteTxnMarkersWhenCorrectMagicVersion_allowedWithAlterCluster(): Unit = {
```

Comment:
nit: we typically don't use underscores in method names like this. Can we stick to camel case here?
Also can we parameterize this test?

Reply:
Made the changes.